infinitest-intellij/src/main/java/org/infinitest/intellij/idea/IdeaModuleSettings.java

```diff
@@ -88,14 +89,26 @@ public RuntimeEnvironment getRuntimeEnvironment() {
 	 * 
 	 * @return A list of all of the output directories for the project
 	 */
-	private List<File> listOutputDirectories() {
-		List<File> outputDirectories = new ArrayList<File>();
+    private List<File> listOutputDirectories() {
+        List<File> outputDirectories = new ArrayList<File>();
+
+        addOutputDirectory(outputDirectories, CompilerPaths.getModuleOutputPath(module, false));
```

Comment:
You could extract `true/false` to constants to improve readability of this part (or maybe there are already corresponding constants in Idea SDK?)